src/mlpack/tests/main_tests/decision_tree_test.cpp

```diff
@@ -152,6 +152,11 @@ BOOST_AUTO_TEST_CASE(DecisionTreeMinimumLeafSizeTest)
 
   // Initialize an all-ones weight matrix.
   arma::mat weights(1, labels.n_cols, arma::fill::ones);
+ 
+  // Input training data.
+  SetInputParam("training", std::move(inputData));
```

Comment:
The test now throws a FATAL error because the input parameter training is supposed to be a tuple containing the info and the inputData.

Reply:
oops! I overlooked changes made by @haritha1313